supabase/migrations/20230525200130_add_project_columns.sql

```diff
@@ -0,0 +1,11 @@
+alter table public.projects 
+add COLUMN "owner" char(42) not null,
+add COLUMN "creator" char(42) not null,
+add COLUMN "contributors_count" int not null,
+add COLUMN "redeem_count" int not null,
+drop column "total_paid",
```

Comment:
this column was just renamed to "volume". `rename` not working, so we just drop and add the new "volume" column